tests/dummy/app/templates/modules/accordion.hbs

```diff
@@ -13,7 +13,7 @@
       <div class="ui hidden divider"></div>
 
       <p>
-        For additional documentation on how to configure this module, please refer to the <a href="/">Getting Started</a> section and Semantic UI's module specific external documentation links below.
+        For additional documentation on how to configure this module, please refer to the <a href="http://semantic-org.github.io/Semantic-UI-Ember/#/modules">Getting Started</a> section and Semantic UI's module specific external documentation links below.
```

Comment:
Thanks for this! I didn't realize I missed that. 
One small change, lets use link-to to ensure if the documentation changes in the future the urls work. Just change all the urls to `{{#link-to 'modules.index'}}Getting Started{{/link-to}}`